### DISCUSSION TITLE
Is Magick.NET-Q16-AnyCPU (13.10.0) library under Apache 2.0 license or under GPL 3.0?

### DISCUSSION THREAD

**Discussion by emp-ypadilla** [dlemstra/Magick.NET#1698] [General]
While do compliance scan it was reported the use of GPL on 4 files.
Notice.osx.txt
Notice.linux.txt
Notice.linux-musl.txt
Notice.win.txt

In case this is true, is there a commercial license on this product?

**Comment by dlemstra** [comment]
There are no GPL licenses in those files only LGPL licenses and those allow commercial use.

**Reply by emp-ypadilla** [reply]
Thank you so much for your time. The issue we are having with the legal department is that there are  multiple reference to components under GPL license in the notice.txt files
For example pango:
https://github.com/dlemstra/Magick.NET/blob/51fd65be461af48ff1ca8901c22439561f1d74f1/src/Magick.Native/libraries/win/Notice.txt#L5428

**Comment by dlemstra** [comment]
That says `GNU LIBRARY GENERAL PUBLIC LICENSE` and that means GNU L (library) G (general) P (public) L (license). More info about that license can be found here: https://fossa.com/blog/open-source-software-licenses-101-lgpl-license/.
